app/src/main/java/com/burhanrashid52/photoediting/EditImageActivity.kt

```diff
@@ -183,20 +184,32 @@ class EditImageActivity : BaseActivity(), OnPhotoEditorListener, View.OnClickLis
         })
     }
 
-    override fun onAddViewListener(viewType: ViewType?, numberOfAddedViews: Int) {
-        Log.d(TAG, "onAddViewListener() called with: viewType = [$viewType], numberOfAddedViews = [$numberOfAddedViews]")
+    override fun onAddViewListener(rootView: View?, viewType: ViewType?, numberOfAddedViews: Int) {
+        val viewRect = Rect()
+        rootView?.getGlobalVisibleRect(viewRect)
```

Comment:
Not sure what `getGlobalVisibleRect` does here, but it would be really helpful if you add an example of your use case here in a more general way like editing the view, adding links, and clicking links in this sample app.
This will also help us to understand how to us these new APIs.

Reply:
For example, check `app/src/main/java/com/burhanrashid52/photoediting/EditImageActivity.java` in #358

Reply:
`getGlobalVisibleRect` is used to get the position of the `rootView` on the screen, it is important since we need to save the exact coordinates of the view to recreate it (if we removed it before saving) or to add a clickable ghost above it. A use-case could be a story like instagram, with a clickable tag or link or location.

I can add such use-case to the sample but it would take some time, maybe for a future collaboration:)